### PR TITLE
Added a new quick references page and a FAQs page

### PR DESCRIPTION
**Pull request by camelCasedAditya**
The quick reference page is for users who already know the basics of how MooseJS works but need to refer to the docs quickly.

### CODE REVIEW

File: apps/framework-docs/src/pages/quick-reference.mdx [hunk 1]
```diff
@@ -0,0 +1,341 @@
...
+        and add a new Data Model definition.
+
+        ```ts filename="datamodels/models.ts" copy
+        import { Key } from "@514labs/moose-lib";
+
+        export interface UserActivity {
+        eventId: Key<string>;
+        timestamp: string;
+        userId: string;
+        activity: string;
```
Comment (callicles) [inline apps/framework-docs/src/pages/quick-reference.mdx]: Looks like you could add some indentation to those code blocks to make them render better

**Review by callicles — COMMENTED** [review]
Small change :)

File: apps/framework-docs/src/pages/quick-reference.mdx [hunk 2]
```diff
@@ -0,0 +1,341 @@
...
+        ```
+
+        ## Supported Types
+
+        Here is the status of MooseJS support for various data types in Data Models:
+        | Clickhouse | Typescript | Moose |
+        |:---------- |:---------- |:------- |
+        | String | String | ✅ |
+        | Boolean | Boolean | ✅ |
+        | Int64 | Number | ✅ |
```
Comment (phiSgr) [inline apps/framework-docs/src/pages/quick-reference.mdx:35]: I think it's misleading to say we support other number types as we map all ts number types to Float64

File: apps/framework-docs/src/pages/faqs.mdx [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# FAQs
+
+---
+
+### Why is Docker is not working after making changes to some fields?
+
+Terminate the Docker containers and restart Docker manually. If you have problems restarting run `npm run dev` twice.
+
+### Why has a test run crashed?
```
Comment (DatGuyJonathan) [inline apps/framework-docs/src/pages/faqs.mdx]: This gives the impression that there's a moose test framework they can run, which we don't yet have. Is this referring to sending data to ingestion point?

**Review by callicles — APPROVED** [review]

File: apps/framework-docs/src/pages/quick-reference.mdx (hunk 2 above)
Comment (camelCasedAditya) [reply apps/framework-docs/src/pages/quick-reference.mdx:35]: I have just added a disclaimer to the quick reference page and the data models setup page in the latest Commit.

File: apps/framework-docs/src/pages/faqs.mdx (hunk 3 above)
Comment (camelCasedAditya) [reply apps/framework-docs/src/pages/faqs.mdx]: I have rephrased this question for more clarity in the most recent commit. Let me know if it sounds better.

**Pull request merged**